app/Position.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Position extends Model
+{
+
+    protected $guarded = [];
```

Comment:
I think we should caution the use of this. Albeit an accepted practice in the Laravel community, we need to be conscious that it means we need to be quite tight in our validation.

Reply:
Agreed!